lib/bokken_web/controllers/file_controller.ex

```diff
@@ -14,4 +17,39 @@ defmodule BokkenWeb.FileController do
   def images(conn, %{"type" => type, "id" => id, "file" => image}) do
     send_file(conn, 200, "uploads/#{type}/#{id}/#{image}")
   end
```

Comment:
It's missing an endpoint for sending project files. Something that could serve the files them selfs.